recipe/meta.yaml

```
@@ -6,39 +6,52 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/langgraph_checkpoint-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
   sha256: 2b800195532d5efb079db9754f037281225ae175f7a395523f4bf41223cbc9d6
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
 
 requirements:
   host:
-    - python {{ python_min }}
-    - poetry-core
+    - python 
     - pip
+    - poetry-core
   run:
+    - python 
     - ormsgpack >=1.8.0,<2.0.0
-    - msgpack-python >=1.1.0,<2.0.0
-    - python >={{ python_min }}
     - langchain-core >=0.2.38,<0.4
```

The `0.4` part applies to python, no? https://github.com/langchain-ai/langgraph/blob/checkpoint%3D%3D2.0.26/libs/checkpoint/pyproject.toml#L13

```suggestion
    - langchain-core >=0.2.38
```

---

done, thank you